airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/StandardSourceTest.java

```diff
@@ -98,16 +98,15 @@
   private Set<String> IMAGES_TO_SKIP_SECOND_INCREMENTAL_READ = Sets.newHashSet(
       "airbyte/source-intercom-singer",
       "airbyte/source-exchangeratesapi-singer",
-      "airbyte/source-hubspot-singer",
+      "airbyte/source-hubspot",
```

Comment:
Is the reason we include this here is because the cursor comparison logic is inclusive? 

how can we verify that incremental is working? can we compare cursor values in a custom test?

Reply:
yes it is inclusive as we use `startTimestamp` param to filter records

Reply:
we could verify that we don't produce records older than the state value in custom test

Reply:
can we do that?

Reply:
I have added a custom test for incremental, unfortunately, `subscription_changes` stream doesn't have data, and all my tries to subscribe and unsubscribe from emails didn't trigger any event there. I remember I was using demo credentials to test development, but obviously, demo creds can't be used in the test, as they mostly give you a random response

Reply:
no problem.